### ISSUE TITLE
move unsafe not generated properly if written in lower-case

### ISSUE THREAD

**Issue by smedilol** [TypeCobolTeam/TypeCobol#357]
`move UNSAFE a to b` is correctly generated
`move unsafe a to b` is not

The quick and dirty fix is to add the following pattern in our skeleton:
```
<skeleton name="UNSAFE">
    <conditions>
      <condition node="TypeCobol.Compiler.CodeElements.VariableWriter" unsafe="true"/>
    </conditions>
    <patterns>
      <pattern location="NODE" action="erase">UNSAFE</pattern>
      <pattern location="NODE" action="erase">unsafe</pattern>
    </patterns>
  </skeleton>
```

However, I think action "Erase" should take an insensitive text.
@wiztigers what do you think?

**Comment by wiztigers** [comment]
I only use quick and dirty solutions to spare a lot of time, and when deadline rings.
This issue match neither description, so I fixed it in 3a00660006422c9883aec14548d041e9167d02e4 (with a little improvement on top).

**Comment by smedilol** [comment]
This fix doesn't works.
